manifests/repos.pp

```diff
@@ -7,15 +7,16 @@
 
   case $::osfamily {
     'Debian': {
-      include apt
-      # apt-transport-https is required by the apt to get the sources
-      ensure_packages(['apt-transport-https'])
-      Package['apt-transport-https'] -> Apt::Source <||>
-      if $::operatingsystem == 'Debian' and $::lsbdistcodename == 'wheezy' {
-        include apt::backports
-      }
-      Exec['apt_update'] -> Package[$docker::prerequired_packages]
       if ($docker::use_upstream_package_source) {
```

Comment:
Wouldn't it be a better approach to check for `if $docker::manage_package` instead? At least this is what I've done locally and what can be found in many other puppet modules as well.